### PR TITLE
Add Nord Pool integration

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

SSIA

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/129983
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/6074
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced comprehensive documentation for integrating Nord Pool energy market prices into Home Assistant.
	- Supports configuration for multiple market areas and currency selection, defaulting to EUR.
	- Added various sensors for current, previous, and block prices, including options for daily averages and diagnostics.

- **Documentation**
	- Included an example configuration demonstrating practical application within Home Assistant.

### CODE REVIEW

File: source/_integrations/nordpool.markdown [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+---
+
+The **Nord Pool** {% term integration %} integrates [Nord Pool Group](https://www.nordpoolgroup.com/) energy prices into Home Assistant.
+
+The {% term integration %} provides the public market prices displayed on the [Nord Pool Auction page](https://data.nordpoolgroup.com/auction/day-ahead/prices).
+
+{% include integrations/config_flow.md %}
+
+{% tip %}
+Only a single configuration entry is supported so ensure you select all the areas of interest when you setup the integration.
```
Comment (gjohansson-ST) [inline source/_integrations/nordpool.markdown]:
```suggestion
Only a single configuration entry is supported, ensure you select all the areas of interest when you set up the integration.
```

File: source/_integrations/nordpool.markdown [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+{% configuration_basic %}
+Areas:
+  description: Select one or multiple of market areas to create sensors for.
+Currency:
+  description: Currency to display prices in, EUR is the base currency in Nord Pool prices.
+{% endconfiguration_basic %}
+
+## Sensors
+
```
Comment (gjohansson-ST) [inline source/_integrations/nordpool.markdown:40]:
```suggestion

Each market area will create a device which has the following sensors:

```

File: source/_integrations/nordpool.markdown [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+### Block price sensors
+
+- Block average
+- Block minimum
+- Block maximum
+- Block start time
+- Block end time
+
+These sensors show the minimum/maximum and average during certain blocks of the day. More known as off-peak (cheaper) or peak hours (expensive).
```
Comment (gjohansson-ST) [inline source/_integrations/nordpool.markdown]:
```suggestion
These sensors show the minimum/maximum and average during certain blocks of the day. More known as off-peak (typically lower price) or peak hours (typically higher price).
```

File: source/_integrations/nordpool.markdown [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+These sensors are not enabled by default and needs to be enabled to be used.
```
Comment (gjohansson-ST) [inline source/_integrations/nordpool.markdown]:
```suggestion
The block price sensors are not enabled by default.
```

File: source/_integrations/nordpool.markdown [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+### Daily average
+
+- Daily average
+
+This sensor is not enabled by default.
```
Comment (gjohansson-ST) [inline source/_integrations/nordpool.markdown]:
```suggestion
The daily average sensor is not enabled by default.
```

File: source/_integrations/nordpool.markdown [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+template:
+  - sensor:
+      - name: "Full SE3 current price"
+        unit_of_measurement: "SEK/kWh"
+        state_class: measurement
+        state: >
+          {% set cost = states('sensor.nord pool_se3_current_price') | float(0) %}
+          {% set add_cost = states('input_number.add_fixed_cost') | float(0) %}
+
+          {{ ((cost + add_cost) * 1.25) | round(2, default=0) }}
```
Comment (gjohansson-ST) [inline source/_integrations/nordpool.markdown]:
```suggestion
          # Add fixed cost to the spot price and add VAT (25%)
          {{ ((cost + add_cost) * 1.25) | round(2, default=0) }}
```

**Comment by gjohansson-ST** [comment]
@coderabbitai review

File: source/_integrations/nordpool.markdown [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+---
+
+The **Nord Pool** {% term integration %} integrates [Nord Pool Group](https://www.nordpoolgroup.com/) energy prices into Home Assistant.
+
+The {% term integration %} provides the public market prices displayed on the [Nord Pool Auction page](https://data.nordpoolgroup.com/auction/day-ahead/prices).
+
+{% include integrations/config_flow.md %}
+
+{% tip %}
+Only a single configuration entry is supported, ensure you select all the areas of interest when you set up the integration.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
Only a single configuration entry is supported. When setting up the integration, ensure you select all the areas of interest.
```

File: source/_integrations/nordpool.markdown [hunk 8]
```diff
@@ -0,0 +1,95 @@
...
+
+EUR is the base currency for market prices, You will find in the `Exchange rate` sensor the relevant conversion used if you choose another currency.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
EUR is the base currency for market prices. If you choose another currency, you can find the conversion rate in the `Exchange rate` sensor.
```

File: source/_integrations/nordpool.markdown [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+All prices will be displayed as `selected_currency/kWh`.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
All prices are displayed as `selected_currency/kWh`.
```

File: source/_integrations/nordpool.markdown [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+{% endtip %}
+
+{% configuration_basic %}
+Areas:
+  description: Select one or multiple of market areas to create sensors for.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
  description: Select one or multiple market areas to create sensors for.
```

File: source/_integrations/nordpool.markdown [hunk 11]
```diff
@@ -0,0 +1,95 @@
...
+Currency:
+  description: Currency to display prices in, EUR is the base currency in Nord Pool prices.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
  description: Currency to display prices in. EUR is the base currency in Nord Pool prices.
```

File: source/_integrations/nordpool.markdown [hunk 12]
```diff
@@ -0,0 +1,95 @@
...
+
+Each market area will create a device which has the following sensors:
+
+### Main sensors
+
+- Current price for the selected area
+- Previous price for the selected area
+- Next price for the selected area
+
+These sensors can be used to calculate your current cost of energy, should I charge my battery now or next hour etc.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
These sensors can be used to calculate your current energy cost, or to help decide whether to charge the battery now or in an hour, etc.
```

**Review by c0ffeeca7 — COMMENTED** [review]
some style tweaks. feel free to rephrase

File: source/_integrations/nordpool.markdown [hunk 13]
```diff
@@ -0,0 +1,95 @@
...
+
+### Daily average
+
+- Daily average
+
+The daily average sensor is not enabled by default.
+
+### Diagnostic sensors
+
+- Last updated - indicated when the market price was last updated.
```
Comment (c0ffeeca7) [inline source/_integrations/nordpool.markdown]:
```suggestion
- Last updated - indicates when the market price was last updated.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @gjohansson-ST :+1:

**Pull request merged**